### PR TITLE
Feature delete bill

### PR DESCRIPTION
**Pull request by at-ngatran**
I have finished function delete bill, please check for me.

### CODE REVIEW

File: app/Http/Controllers/BillController.php [hunk 1]
```diff
@@ -94,4 +94,28 @@ public function show($id)
...
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        $errors = trans('bills.delete.error_message');
+        try {
+            $bill = Bill::findOrFail($id);
+            $billDetails = $bill->billDetails;
+            foreach ($billDetails as $billDetail) {
```
Comment (dzungnguyenh) [inline app/Http/Controllers/BillController.php]: [rank A] please investigate about transaction in Laravel and batch delete. We should avoid use a loop to execute many queries.

File: resources/views/statistics/daily.blade.php [hunk 2]
```diff
@@ -18,152 +18,35 @@
...
-                            </tr>
-                          @endforeach
-                        </tbody>
-                      </table>
-                @endif
+            <div class="row">
+              <div class="col-md-6 col-sm-6 col-xs-12">
+                <div class="x_panel">
+                  <div class="x_title">
+                    <h2>Line graph<small>Sessions</small></h2>
```
Comment (dzungnguyenh) [inline resources/views/statistics/daily.blade.php]: [rank C] please put this to language file or something like that.

**Comment by at-ngatran** [comment]
I have fixed delete bill, please check for me. Thanks

**Comment by dzungnguyenh** [comment]
Good!. Thanks.

**Pull request merged**